README.md

```diff
@@ -166,6 +166,14 @@ If you want to limit memory to 64GB but allow MarkLogic Server to use swap space
 ```
 To allow MarkLogic Server to use unlimited memory and swap space (up to the amount available on the host system), do not specify either "--memory" or "--memory-swap".
 
+## Configuring HugePages
+
```

Comment:
Sumanth, shouldn't this section and the two before it be under "Configuration" not "Using This Image"? I suggest moving them under "Configuration".

Reply:
Moved it. Good point